db/migrate/20171019115705_add_classification_export_row_model.rb

```diff
@@ -0,0 +1,27 @@
+class AddClassificationExportRowModel < ActiveRecord::Migration
+  def change
+    create_table :classification_export_rows do |t|
+      t.references :classification, null: false
+      t.references :project, null: false
+      t.references :workflow, null: false, index: true
+      t.references :user
+      t.string :user_name
+      t.string :user_ip
+      t.string :workflow_name
+      t.string :workflow_version
+      t.timestamp :classification_created_at
+      t.boolean :gold_standard
+      t.string :expert
+      t.jsonb :metadata
+      t.jsonb :annotations
+      t.jsonb :subject_data
+      t.string :subject_ids
+      t.timestamps
+    end
+
+    add_index :classification_export_rows,
+      %i(project_id classification_id),
+      name: 'index_export_rows_on_project_id_and_classification_id',
+      unique: true
```

Comment:
Aren't classification IDs globally unique? Why include the project in this unique index?

Reply:
they are unique, i want the project index to increase how selective this index when we run a dump for backfilling. Finding can probably just use the project_id / workflow_id index but i should check the `find_in_batches` won't use the compound index to order by classification (i think it does by default).